projects/binary-dependency/genezio-runtime-linux.yaml

```
@@ -0,0 +1,19 @@
+name: binary-dependency
+region: us-east-1
+yamlVersion: 2
+backend:
+    path: ./server
+    language:
+        name: js
+        runtime: nodejs18.x
+        packageManager: npm
+    cloudProvider: capybaraLinux
+    scripts:
+        deploy:
+            - npm install
```

Can you go through all YAMLs and use array only if there are more than 1 commands?
```suggestion
        deploy: npm install
```